roles/ceph-config/templates/ceph.conf.j2

```diff
@@ -171,6 +171,7 @@ host = {{ hostvars[host]['ansible_hostname'] }}
 host = {{ hostvars[host]['ansible_hostname'] }}
 keyring = /var/lib/ceph/radosgw/{{ cluster }}-rgw.{{ hostvars[host]['ansible_hostname'] }}/keyring
 log file = /var/log/ceph/{{ cluster }}-rgw-{{ hostvars[host]['ansible_hostname'] }}.log
+rgw keystone implicit tenants = true
```

Comment:
we don't accept PR on this file anymore, unless there is a case where `ceph_conf_overrides` can not be used. Please use `ceph_conf_overrides`.